src/main/resources/default/taglib/t/deleteButton.html.pasta

```diff
@@ -1,11 +1,13 @@
-<i:arg type="String" name="url"/>
-<i:arg type="Page" name="page" default=""/>
+<i:arg name="url" type="String" description="Specifies the url which should be called. Is only used when page is left empty."/>
```

Comment:
Have a look at line 11 where the HREF attribute is generated. The URL is used even if there is a `page`.